fedimint-cli/src/envs.rs

```diff
@@ -5,6 +5,9 @@ pub const FM_CLIENT_DIR_ENV: &str = "FM_CLIENT_DIR";
 // Env variable to set the peer id of the guardian
 pub const FM_OUR_ID_ENV: &str = "FM_OUR_ID";
 
+// Env variable to set the connector choice (e.g Tcp / Tor)
+pub const FM_CONNECTOR_ENV: &str = "FM_CONNECTOR";
```

Comment:
I don't know if the "connector" thing is a good way to go about it in the UI. Maybe we should just have `--use-tor` and `FM_USE_TOR`  or something?

Reply:
Sure, that looks like a better approach.

Reply:
env is nice to have

Reply:
I updated it to expect the `--use-tor` and kept the `FM_USE_TOR` environment variable too.